atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/CandidateCellForSweeping.java

```diff
@@ -41,11 +41,4 @@
      * Otherwise, the return value is undefined and depends on the implementation.
      */
     boolean isLatestValueEmpty();
-
-    static CandidateCellForSweeping of(Cell cell, Collection<Long> sortedTimestamps) {
```

Comment:
Totally unused, both internally and externally (and just wrong, since it doesn't set `isLatestValueEmpty`).